packages/backend/src/entity/mongoRepository.ts

```diff
@@ -43,11 +43,15 @@ async function findByDid(collection: Collection, did: string) {
   return result && Entity.parse(result)
 }
 
-async function addEntity(collection: Collection, entity: Entity) {
+async function addEntity(
+  collection: Collection,
+  entity: Entity,
+  config?: { session?: ClientSession },
```

Comment:
I'm not sure about using `config` as the name of a function parameter in this case. What's the meaning of that? Is it a config of an entity, app, or something else? What if we use just `session`?

Reply:
`session` is fine, now that you pointed out that `newDatestamps` isn't necessary in this PR.
`config` was just an attempt to structure the data if there were multiple params.